tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+in the right window showing connection url (screenshot below). And click save
+![initial load](4_striim_initial_load.png)
+
+6. Created > Deploy App > Deploy
+![initial load deploy](5_striim_deploy.png)
+
+7. Deploy > Start App. 
+8. You will soon observe rows flowing through striim and data being inserted inside the cloud spanner.
+9. On spanner gui verify that rows have been written with bit reversed id values.
+![initial complete](6_spanner_il_complete.png)
+
+### Create continuous replication (CDC) pipeline
+Once initial load is complete, then using CDC pipeline you can keep replicating the new changes into Spanner.  
+Keeping MySQL and Spanner in sync. It reads data from mysql's binary logs using Striim's MysqlReader component.  
+
+In production you would use `StartTimestamp` property to specify binary log position (i.e. time at which initial load started).  
+Also, you might like to create a `CheckpointTable` so that [striim can recover](https://www.striim.com/docs/en/spanner-writer.html) from a crash.
+However these concepts are out of scope for this tutorial.
+
+1. Similar to previous, import below code for CDC Pipeline and change Primary_IP with mysql’s ip. Then deploy and start the app.
+
+        drop APPLICATION mysql_to_spanner_cdc CASCADE;
+        
+        IMPORT STATIC CustomFunctions.*;
+        
+        CREATE APPLICATION mysql_to_spanner_cdc;
+        
+        CREATE FLOW sample1_cdc;
+        
+        
+        CREATE SOURCE mysql_cdc_source USING MysqlReader  (
+        ConnectionURL: 'jdbc:mysql://PRIMARY_IP:3306/employeedb',
+          Username: 'root',
+          Compression: false,
+          Password_encrypted: 'false',
+          connectionRetryPolicy: 'retryInterval=30, maxRetries=3',
+          FilterTransactionBoundaries: true,
+          Tables: 'employeedb.signin_log',
+          Password: 'password123',
+          SendBeforeImage: true
+          --change this timestamp
+          --,StartTimestamp: '2021-MAR-19 13:00:00'
+          )
+        OUTPUT TO sample_raw_cdc;
+        
+        
+        CREATE STREAM SampleModifiedDataCDC OF Global.WAEvent;
+        
+        CREATE CQ POPULATE_SAMPLE_TABLE_CDC
+        INSERT INTO SampleModifiedDataCDC
+        SELECT * FROM sample_raw_cdc
+        MODIFY(data[0] = bitReverseInt(data[0]));
+        
+        
+        CREATE TARGET spanner_sample_cdc USING Global.SpannerWriter (
+          InstanceID: 'spanner-tgt',
+          --CheckpointTable: 'CHKPOINT',
+          BatchPolicy: 'EventCount: 100, Interval: 10s',
+          ParallelThreads: '',
+          ServiceAccountKey: '/opt/striim/striim-spanner-key.json',
+          Tables: 'employeedb.signin_log,employeedb.signin_log' )
+        INPUT FROM SampleModifiedDataCDC;
+        
+        END FLOW sample1_cdc;
+        
+        END APPLICATION mysql_to_spanner_cdc;
+
+2. Connect to mysql and insert a few rows.
```

Comment:
Should we link or add a statement on how to connect to MySQL?

Reply:
Added connection command.